proto/internalapi/central/image.proto

```diff
@@ -15,3 +17,10 @@ message ScanImage {
     // force will cause central and sensor caches to be ignored.
     bool   force      = 3;
 }
+
+// ImageIntegrations contains a list of integrations sensor should upsert and/or
+// delete from its internal store. 
+message ImageIntegrations {
+    repeated storage.ImageIntegration updated_integrations    = 1;
```

Comment:
is this all integrations or just "new" ones?

Reply:
Depends, when sensor first establishes a connection it will be all integrations, after that the msg will be sent for any integration thats added, updated, or deleted.